test/integration/user_visits_place_test.rb

```diff
@@ -26,18 +26,35 @@ class UserVisitsPlaceSpec < ActionDispatch::IntegrationTest
     VCR.use_cassette("existing place") do
       visit "/"
 
-      page.fill_in 'nav-search', 
+      page.fill_in 'nav-search',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.